app/services/workflow_service/disposable_capital_assessment.rb.orig

```diff
@@ -0,0 +1,50 @@
+module WorkflowService
+  class DisposableCapitalAssessment < BaseWorkflowService
+<<<<<<< HEAD
```

Comment:
Looks like something has gone wrong with a merge here.

Reply:
In fact it looks like this whole file shouldn't have been committed. Should .orig files be added to .gitignore?